music_assistant/providers/alexa/__init__.py

```diff
@@ -0,0 +1,408 @@
+"""
+DEMO/TEMPLATE Player Provider for Music Assistant.
```

Comment:
I think should probably make sure all the comments are appropriate to this provider. That is, remove all these generic comments or replace.

Reply:
I think the comments just need to appropriate to the code